File: app/src/main/java/io/ipoli/android/quest/widgets/AgendaWidgetService.java
package io.ipoli.android.quest.widgets;

import android.content.Intent;
import android.widget.RemoteViewsService;

/**
 * Created by Venelin Valkov <venelin@curiousily.com>
 * on 5/18/16.
 */
public class AgendaWidgetService extends RemoteViewsService {

    @Override
    public RemoteViewsFactory onGetViewFactory(Intent intent) {
        return new QuestRemoteViewsFactory(getApplicationContext());
    }
}

File: app/src/main/java/io/ipoli/android/quest/widget/agenda/AgendaWidgetService.kt
package io.ipoli.android.quest.widget.agenda

/**
 * Created by Venelin Valkov <venelin@mypoli.fun>
 * on 02/10/2018.
 */

import android.content.Intent
import android.widget.RemoteViewsService

class AgendaWidgetService : RemoteViewsService() {

    override fun onGetViewFactory(intent: Intent?) =
        AgendaWidgetViewsFactory(applicationContext)
}